hermes/views.py

```diff
@@ -78,19 +87,63 @@ def validate_brightness_unit(self, data):
             if row['brightness_unit'] not in brightness_units:
                 raise ValidationError(f'Unrecognized brightness unit. Accepted brightness units are {brightness_units}')
 
-
-class CandidateMessageSubSchema(Schema):
-    authors = fields.String()
-    main_data = fields.Nested(CandidateDataSchema)
-
-
-class CandidateMessageSchema(Schema):
+class MessageSchema(Schema):
     title = fields.String(required=True)
     topic = fields.String(required=True)
     event_id = fields.String()
     message_text = fields.String(required=True)
-    author = fields.String(required=True)
-    data = fields.Nested(CandidateMessageSubSchema)
+    submitter = fields.String(required=True)
+    authors = fields.String()
+    @property
+    @abstractmethod
+    def data(self):
+        pass
+
+
+class CandidateMessageSchema(MessageSchema):
+    data = fields.Nested(CandidateDataSchema)
+
+
+class PhotometryDataSchema(Schema):
+    target_name = fields.String()
+    ra = fields.String(required=True)
+    dec = fields.String(required=True)
+    date_observed = fields.String(required=True)
+    date_format = fields.String()
```

Comment:
How are we going to handle different reference systems?  E.g. in exoplanet transits it is common to quote Barycentric Julian Date rather than Heliocentric Julian Date or Modified Julian Date, let alone all the different Gregorian date formats.

Reply:
Ah I see below this is parsed as an astropy.time.Time object.  That's fine but I wonder if it would be worth including the date system as a separate field as well?